frontend/src/components/ItemMetadataForm.md

```diff
@@ -39,5 +39,5 @@ If you want to show an error state, set the errors object:
 const item = { title: 'The Dead Parrot Sketch' };
 const errors = { title: ['The parrot is alive.', 'It is sleeping.'] };
 
-<ItemMetadataForm item={ item } errors={ errors } />
+<ItemMetadataForm item={ item } errors={ errors } onChange={ _ => {} } />
```

Comment:
I think the correct fix is to make onChange not required and either
1. fix up the call sites so ``event => onChange(...)`` becomes ``event => onChange && onChange(...)``, or
2. give onChange the default value of ``() => null``.